crates/rbuilder/src/live_builder/base_config.rs

```diff
@@ -284,7 +284,17 @@ impl BaseConfig {
     }
 
     pub fn coinbase_signer(&self) -> eyre::Result<Signer> {
-        coinbase_signer_from_secret_key(&self.coinbase_secret_key.value()?)
+        if let Some(secret_key) = &self.coinbase_secret_key {
+            if let Ok(resolved_key) = secret_key.value() {
```

Comment:
```suggestion
           coinbase_signer_from_secret_key(secret_key.value()?)
```
We should fail if parsing the user-supplied secret key fails.